ckanext/dcatapchharvest/profiles.py

```diff
@@ -892,6 +897,12 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
                 g.add((doc, RDF.type, FOAF.Document))
                 g.add((distribution, FOAF.page, doc))
 
+            # Access Services
+            access_services = resource_dict.get('access_services', [])
+            for uri in access_services:
+                ref = URIRef(uri)
+                g.add((distribution, DCAT.accessService, ref))
```

Comment:
You need to add also 
`g.add((ref, RDF.type, DCAT.accessService))`
to ensure that your `ref` URI is of type DCAT.accessService.

Reply:
No, this is the output that I get with this code:

```
    <dcat:distribution>
      <dcat:Distribution rdf:about="urn:uuid:80b34b3e-aedb-5081-669e-17c23d0d9bf8">
        ...
        <dcat:accessService rdf:resource="https://geoportal.sachsen.de/md/685a4409-a026-430e-afad-1fa2881f9700"/>
        ...
      </dcat:Distribution>
    </dcat:distribution>
```

Reply:
ahh I see, thank you for checking